dotcom-rendering/cypress/e2e/parallel-7/snapshot-standard-layout.e2e.cy.js

```diff
@@ -0,0 +1,23 @@
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+import { breakpoints } from '@guardian/source-foundations';
+import '@percy/cypress';
+
+describe('Snapshot standard layout', function () {
+	beforeEach(function () {
+		disableCMP();
+		setLocalBaseUrl();
+	});
+
+	it('Snapshot all breakpoints', function () {
+		cy.visit(
+			'/Article?url=https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents',
+		);
```

Comment:
Is the intention here to map the possible `Design` and `Display` variations, snapshotting each one?

Reply:
I'd suggest we snapshot important variations although overlap with Chromatic will need to considered. 

@bryophyta if you don't mind I think you are better placed to answer this - thanks.

Reply:
> overlap with Chromatic

Is the goal to remove our layout tests in Chromatic and replace them with these?

Reply:
It feels like there are still some unknowns around how this might work in practice, but it seems to me that this would be a positive move if we can make it work. Shall we make some time to pair on this next week?

Reply:
Sounds good 👍  

This shouldn't block this, PR by the way, it was just a question